diff --git a/node_modules/werift/lib/index.mjs b/node_modules/werift/lib/index.mjs
index bd453cf..2e8ebc9 100644
--- a/node_modules/werift/lib/index.mjs
+++ b/node_modules/werift/lib/index.mjs
@@ -12750,7 +12750,7 @@ var log30 = debug("werift/sctp/sctp");
 var COOKIE_LENGTH = 24;
 var COOKIE_LIFETIME = 60;
 var MAX_STREAMS = 65535;
-var USERDATA_MAX_LENGTH = 1200;
+var USERDATA_MAX_LENGTH = 1024;
 var SCTP_DATA_LAST_FRAG = 1;
 var SCTP_DATA_FIRST_FRAG = 2;
 var SCTP_DATA_UNORDERED = 4;
diff --git a/node_modules/werift/lib/sctp/src/sctp.js b/node_modules/werift/lib/sctp/src/sctp.js
index 45d5e3c..9a582c8 100644
--- a/node_modules/werift/lib/sctp/src/sctp.js
+++ b/node_modules/werift/lib/sctp/src/sctp.js
@@ -13,7 +13,7 @@ const log = (0, common_1.debug)("werift/sctp/sctp");
 const COOKIE_LENGTH = 24;
 const COOKIE_LIFETIME = 60;
 const MAX_STREAMS = 65535;
-const USERDATA_MAX_LENGTH = 1200;
+const USERDATA_MAX_LENGTH = 1024;
 // # protocol constants
 const SCTP_DATA_LAST_FRAG = 0x01;
 const SCTP_DATA_FIRST_FRAG = 0x02;
