diff --git a/node_modules/werift-sctp/lib/sctp/src/sctp.js b/node_modules/werift-sctp/lib/sctp/src/sctp.js
index 1bf6928..6c5fba2 100644
--- a/node_modules/werift-sctp/lib/sctp/src/sctp.js
+++ b/node_modules/werift-sctp/lib/sctp/src/sctp.js
@@ -14,7 +14,7 @@ const log = (0, common_1.debug)("werift/sctp/sctp");
 const COOKIE_LENGTH = 24;
 const COOKIE_LIFETIME = 60;
 const MAX_STREAMS = 65535;
-const USERDATA_MAX_LENGTH = 1200;
+const USERDATA_MAX_LENGTH = 1024;
 // # protocol constants
 const SCTP_DATA_LAST_FRAG = 0x01;
 const SCTP_DATA_FIRST_FRAG = 0x02;
